server/plugin/oAuth.go

```diff
@@ -108,6 +108,12 @@ func (p *Plugin) OAuthComplete(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := p.GenerateOAuthToken(code, state); err != nil {
+		if strings.Contains(err.Error(), "already connected") {
```

Comment:
constants for this maybe ?

Reply:
I don't think that will be of more use as this is the only place where we need this string also the string itself is descriptive so creating a constant for readability won't be that needy